lib/main.js

```diff
@@ -121,12 +122,14 @@ exports.main = function(options, callbacks) {
         ss.storage.meanings = true;
     }
 
+    var is_install = false;
```

Comment:
@bsstoner In order to test this out I suggest you change the value of this variable to `true` and open `https://duckduckgo.com` in the "test session". A pixel request should then be visible. I understand that this is not the same as the intended usecase but it is the closest we can get to, given the constrains.